With update from Studio 12.4 to 12.6 yet another Solaris workaround seems
to be needed. See bug 16418361 for one of previous Studio 12.4 issues.

diff --git a/hw/xfree86/utils/cvt/cvt.c b/hw/xfree86/utils/cvt/cvt.c
index c49eb27..ef85a80 100644
--- a/hw/xfree86/utils/cvt/cvt.c
+++ b/hw/xfree86/utils/cvt/cvt.c
@@ -30,6 +30,7 @@
 #include "regionstr.h"
 _X_EXPORT BoxRec RegionEmptyBox = { 0, 0, 0, 0 };
 _X_EXPORT RegDataRec RegionEmptyData = { 0, 0 };
+_X_EXPORT RegDataRec RegionBrokenData = { 0, 0 };
 
 /* added to quieten undefined symbol error while compiling with Studio 12.4 */
 #include "dixstruct.h"
